EarlGrey/Common/GREYAnalytics.m

```diff
@@ -37,35 +38,50 @@
  */
 static NSString *const kTrackingEndPoint = @"https://ssl.google-analytics.com";
 
-/**
- *  The currently set GREYAnalytics delegate for custom handling of analytics.
- */
-Class<GREYAnalyticsDelegate> gAnalyticsDelegate;
-
-@interface GREYAnalytics ()<GREYAnalyticsDelegate>
-@end
+@implementation GREYAnalytics {
+  // Overriden GREYAnalytics delegate for custom handling of analytics.
+  Class<GREYAnalyticsDelegate> _delegate;
```

Comment:
Since this is an instance now please make it an instance var instead of class var.